src/contracts/libraries/SafeMath.sol

```diff
@@ -84,7 +85,22 @@ library SafeMath {
      * - The divisor cannot be zero.
      */
     function div(uint256 a, uint256 b) internal pure returns (uint256) {
-        require(b > 0, "SafeMath: division by zero");
+        require(b > 0, "SafeMath: division by 0");
         return a / b;
     }
+
+    /**
+     * @dev Returns the ceiling integer division of two unsigned integers,
+     * reverting on division by zero. The result is rounded towards up the
+     * nearest integer, instead of truncating the fractional part.
+     *
+     * Requirements:
+     *
+     * - The divisor cannot be zero.
+     * - The sum of the dividend and divisor cannot overflow.
+     */
+    function ceilDiv(uint256 a, uint256 b) internal pure returns (uint256) {
+        require(b > 0, "SafeMath: ceiling division by 0");
+        return add(a, b - 1) / b;
```

Comment:
Nitpicky, but we might as well do
```
return a / b + (a % b == 0) ? 0 : 1;
```
(Note the plus operation doesn't overflow.)
This way we don't need to call `add` and we don't revert if `a + b - 1 > 2**256-1`.